soteria/lib/soteria_std/map.ml

```diff
@@ -1,3 +1,8 @@
+(** A wrapper around [Stdlib.Map] that includes a pretty-printer.
```

Comment:
Let's be consistent between modules.
`list.ml` says "extensions of Stdlib.List` while this says "a wrapper around Stdlib.Map`

I prefer extensions